test/e2e/specs/editor/plugins/post-type-locking.spec.js

```diff
@@ -273,9 +273,15 @@ test.describe( 'Post-type locking', () => {
 				.fill( 'p1' );
 
 			await editor.clickBlockOptionsMenuItem( 'Delete' );
-			await expect.poll( editor.getBlocks ).not.toMatchObject( [
+			await expect.poll( editor.getBlocks ).toMatchObject( [
```

Comment:
Oh, okay I just now understand what you mean in the original code 😅. Probably what the intent is to check that the blocks doesn't contain the paragraph block? This works but we could also do the below if we want to emphasize the intent.

```js
await expect.poll( editor.getBlocks ).not.toContainEqual(
  expect.objectContaining( { name: 'core/paragraph' } )
);
```

Reply:
The current code probably isn't that different from what snapshots were testing, so from a migration PoV it should be good.

But I think we can use this pattern in the future.